test/integration/ledger/node/tx.js

```diff
@@ -172,12 +172,36 @@ const transactions = {
     signedTxCborHex: '83a400818258200a0172847d39d5ecb8ed921130415ba01b6c785651b99e9fe969b7837181bf5b0001818258390014c16d7f43243bd81478e68b9db53a8528fd4fb1078d58d54a7f11241d227aefa4b773149170885aadba30aab3127cc611ddbc4999def61c1a00e1fb90021a0002e6300758207ad89f0443618428fbf1e9024686a20ee284735a993f9e770896c1a0a8c87014a10081825820cd2b047d1a803eee059769cffb3dfd0a4b9327e55bc78aa962d9bd4f720db0b258404cadece6fa2e7bb8a52d7d7e3204e0a0540a28bcd072a0a5ab57967f5d95e3f3b4c11450be7c7b161ed7acef21ec0a11ba645ad7383ed09ea2d112a85da86b0082a1016763617264616e6f80',
     network: 'TESTNET',
   },
-  withMint: {
+  ordinary_withMint: {
     // 3b2af509c0c52f728714afe2ca15c3965c03e775bca3a9279fb24a2e5c746f0d
-    unsignedCborHex: '83a400818258203b40265111d8bb3c3c608d95b3a0bf83461ace32d79336579a1939b3aad1c0b700018182583901eb0baa5e570cffbe2934db29df0b6a3d7c0430ee65d4c3a7ab2fefb91bc428e4720702ebd5dab4fb175324c192dc9bb76cc5da956e3c8dff821a001e8480a1581c0d63e8d2c5a00cbcffbdf9112487c443466e1ea7d8c834df5ac5c425a14874657374436f696e1a0078386202182a09a1581c0d63e8d2c5a00cbcffbdf9112487c443466e1ea7d8c834df5ac5c425a24874657374436f696e1a007838624875657374436f696e3a007838619f8201858200581cc4b9265645fde9536c0795adbcc5291767a0c61fd62448341d7e03868202828200581cc4b9265645fde9536c0795adbcc5291767a0c61fd62448341d7e03868200581c0241f2d196f52a92fbd2183d03b370c30b6960cfdeae364ffabac889830302838200581cc4b9265645fde9536c0795adbcc5291767a0c61fd62448341d7e03868200581c0241f2d196f52a92fbd2183d03b370c30b6960cfdeae364ffabac8898200581ccecb1d427c4ae436d28cc0f8ae9bb37501a5b77bcc64cd1693e9ae2082041864820518c8fff6',
+    unsignedCborHex: '83a5008182582048665d1d06d3ad6671429b3dc47ff73b1f4ceaa3b9052fb4833bc724ac55a3460001818258390014c16d7f43243bd81478e68b9db53a8528fd4fb1078d58d54a7f11241d227aefa4b773149170885aadba30aab3127cc611ddbc4999def61c821a3b980a73a1581cdc0cbdad231110934c2f144587ffbb1537396e8c53be8d2cba4dfd79a14756616375756d731a000f4240021a0002bf8d031a024171a809a1581cdc0cbdad231110934c2f144587ffbb1537396e8c53be8d2cba4dfd79a14756616375756d731a000f42409f8200581c43040068ce85252be6164296d6dca9595644bbf424b56b7424458227fff6',
```

Comment:
Why is this so much shorter?

Reply:
The transaction before was probably generate like this: https://github.com/gabrielKerekes/trezor-tx-signing-test, e.g. it contained much more than minting. This transaction contains only minting, nothing else and is made from an ordinary address.